gendox-core-api/src/main/java/dev/ctrlspace/gendox/gendoxcoreapi/ai/engine/utils/constants/CohereConfig.java

```diff
@@ -0,0 +1,13 @@
+package dev.ctrlspace.gendox.gendoxcoreapi.ai.engine.utils.constants;
+
+public class CohereConfig {
+    public static final String AUTHORIZATION = "Authorization";
+    public static final String BEARER = "Bearer ";
+        public static final String MEDIA_TYPE = "application/json";
+    public static final String COMPLETION_URL = "https://api.cohere.ai/v1/generate";
+    public static final String EMBEDDINGS_URL = "https://api.cohere.ai/v1/embed";
+
```

Comment:
Aren't these coming from DB?

Reply:
These are indeed not form the db(also in openAI adapter). I had told you about that and we agreed not to tackle this right now because I had difficulty in creating the adapter, but that it should change in the future. Should I make a ticket about it?

Reply:
These are indeed not form the db. We had talked about it and we agreed to leave it for later as I had a difficult time with the adapter structure in general. I think this change could take a while should I make a ticket about that?

Reply:
yes, please create a ticket